model/jvm/src/test/resources/explore/model/ObsSummary-u9pv5ogAVP04lI9PYNyoP9w_EECItUaAtJl4c534TuK=.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id" : "o-361857f48a6a4d36",
+  "name" : "돚ꑰ쯁Ღ숅Ⱆ◖䎨ꃧ卙嵐蚨ƨ웥筅ᑤ헒ᖖဩ镾뎻蔳꺀콸뺗搌࠲濹痘྄ 圿অ菜ႇ疮⢓빀ﯱﷁ匊୴늑凇싒興谉X慮⨩㵿杻ꨁᬝ堊雀砑꣫㵃䄖賙ꈭ➂Ō敩ۜ",
+  "observationTarget" : {
+    "type" : "Target",
+    "target_id" : "t-1"
+  },
```

Comment:
Let's make sure that in the golden test cases we have all 3 cases for `observationTarget`: at leas one with an asterism, one with a target, and one with `null`. 

It has taken me a few tries to get this, maybe we want to increase the number of cases.

Reply:
Yeah. I see what you mean. now that we have constraint set, too, we'll probably want to increase the number of cases. I'll do that and make sure we have a mix.